components/scream/ekat/cmake/EkatMpiUtils.cmake

```diff
@@ -33,23 +34,26 @@ macro (DisableMpiCxxBindings)
 
   if ("${DISTRO_NAME}" STREQUAL "openmpi")
     add_definitions (-DOMPI_SKIP_MPICXX)
-  elseif ("${DISTRO_NAME}" STREQUAL "mpich")
+  elseif ("${DISTRO_NAMEi}" STREQUAL "mpich")
```

Comment:
Is the `i` at the end of `DISTRO_NAME` a typo?

Reply:
Yup, good catch. Can you tell whether I use vim or emacs? =)